pkg/services/object_manager/placement/netmap.go

```diff
@@ -45,11 +60,28 @@ func (b *netMapBuilder) BuildPlacement(cnr cid.ID, obj *oid.ID, p *netmapSDK.Pla
 	binCnr := make([]byte, sha256.Size)
 	cnr.Encode(binCnr)
 
+	b.mtx.Lock()
+	if nm == b.lastNm {
```

Comment:
Theoretically we can get same network map with different pointers . Do we wan't to handle this case somehow?

Reply:
No, this is bad, but expected. We could solve this in the cache itself, if needed.